src/Checkbox/tests.jsx

```diff
@@ -49,7 +49,7 @@ describe( 'Checkbox', () =>
         {
             test( 'should be false by default', () =>
             {
-                expect( wrapper.prop( 'isReadOnly' ) ).toBe( false );
+                expect( wrapper.prop( 'isReadOnly' ) ).toBeFalsy();
```

Comment:
Since the test reads `'should be false by default'` we should explicitly test that the value is `false`, not just falsy.